sys/arduino/include/arduino.hpp

```diff
@@ -80,7 +80,7 @@ int digitalRead(int pin);
  *
  * @param[in] msec      number of milliseconds to sleep
  */
-void delay(int msec);
+void delay(uint32_t msec);
```

Comment:
same type issue here: the Arduino API specifies this parameter to `unsigned long` [here](https://www.arduino.cc/en/Reference/Delay)